tests/test_scan.py

```diff
@@ -1,9 +1,17 @@
+import asyncio
+
 import pytest
 
 from aioelasticsearch import NotFoundError
 from aioelasticsearch.helpers import Scan
 
 
+def test_scan_default_loop(es, loop):
```

Comment:
Scan loop was needed only for 3.4 version to create futures, it is not needed anymore at all